CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 - Removed the deprecated [GetBlindedBlock](https://ethereum.github.io/beacon-APIs/#/ValidatorRequiredApi/produceBlindedBlock)
 - Removed the deprecated [GetBlockV2](https://ethereum.github.io/beacon-APIs/?urls.primaryName=dev#/Validator/produceBlockV2)
 - Implemented [PostAggregateAndProofsV2](https://ethereum.github.io/beacon-APIs/?urls.primaryName=dev#/Validator/publishAggregateAndProofsV2) (adding support for Electra)
+- Renamed validator metrics `attestation_publication_delay` and `block_publication_delay` to include the suffix `_total` added by the current version of prometheus.
```

Comment:
i think these end up coming out as like validator_attestation_publication_delay_total...
```
curl http://localhost:8008/metrics |grep publication_delay
...
# TYPE validator_block_publication_delay_total counter
validator_block_publication_delay_total{interval="[500,1000)",} 2.0
# TYPE validator_attestation_publication_delay_total counter
validator_attestation_publication_delay_total{interval="[500,1000)",} 30.0
```

Reply:
and worth adding `beacon_block_import_delay_counter_total`

Reply:
fixed, thanks for picking up those!